forge-gui/res/cardsfolder/upcoming/the_second_doctor.txt

```diff
@@ -0,0 +1,12 @@
+Name:The Second Doctor
+ManaCost:2 W U
+Types:Legendary Creature Time Lord Doctor
+PT:2/4
+S:Mode$ Continuous | Affected$ Player | SetMaxHandSize$ Unlimited | Description$ Players have no maximum hand size.
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigChoose | TriggerDescription$ How Civil of You — At the beginning of your end step, each player may draw a card. Each opponent who does can't attack you or permanents you control during their next turn.
+SVar:TrigChoose:DB$ RepeatEach | RepeatPlayers$ Player | RepeatSubAbility$ DBAllowDraw
+SVar:DBAllowDraw:DB$ Draw | OptionalDecider$ True | Defined$ Remembered | RememberDrawn$ True | NumCards$ 1 | SubAbility$ DBEffect 
+SVar:DBEffect:DB$ Effect | RememberObjects$ Remembered | ConditionDefined$ Remembered | ConditionPresent$ Card | StaticAbilities$ STCantAttack | Duration$ UntilYourNextTurn | SubAbility$ DBCleanup
```

Comment:
probably easiest if you give the effect to the opponents so the Duration can be adjusted seamlessly